test/core/channel/channelz_registry_test.cc

```diff
@@ -54,6 +54,7 @@ TEST(ChannelzRegistryTest, UuidStartsAboveZeroTest) {
 TEST(ChannelzRegistryTest, UuidsAreIncreasing) {
   int object_to_register;
   std::vector<intptr_t> uuids;
+  uuids.reserve(10);
```

Comment:
nit: you can accomplish this in a single line: `std::vector<intptr_t> uuids(10);`

Reply:
Done

Reply:
I also fixed a few more instances of this